docs/configuration-reference/platforms/aws.md

```diff
@@ -219,10 +221,11 @@ worker_pool "my-worker-pool" {
 | `region`                      | AWS region to use for deploying the cluster.                                                                                                                                               | "eu-central-1"  |    string    |  false   |
 | `enable_aggregation`          | Enable the Kubernetes Aggregation Layer.                                                                                                                                                   |      true       |     bool     |  false   |
 | `enable_tls_bootstrap`        | Enable TLS bootstraping for Kubelet.                                                                                                                                                       |      true       |     bool     |  false   |
+| `encrypt_pod_traffic`         | Enable in-cluster pod traffic.                                                                                                                                                             |      false      |     bool     |  false   |
 | `disk_size`                   | Size of the EBS volume in GB.                                                                                                                                                              |       40        |    number    |  false   |
 | `disk_type`                   | Type of the EBS volume (e.g. standard, gp2, io1).                                                                                                                                          |      "gp2"      |    string    |  false   |
 | `disk_iops`                   | IOPS of the EBS volume (e.g 100).                                                                                                                                                          |        0        |    number    |  false   |
-| `network_mtu`                 | CNI interface MTU. Use 8981 if using instances types with Jumbo frames.                                                                                                                    |      1480       |    number    |  false   |
+| `network_mtu`                 | CNI interface MTU. Use 1440 if `encrypt_pod_traffic = true`. Use 8981 if using instances types with Jumbo frames and `encrypt_pod_traffic = false` else use 8941.                          |      1480       |    number    |  false   |
```

Comment:
```suggestion
| `network_mtu`                 | CNI interface MTU. Use 1440 if `encrypt_pod_traffic = true`. When using instance types with Jumbo frames, use 8981 (or 8941 if `encrypt_pod_traffic = true`).                          |      1480       |    number    |  false   |
```

Reply:
I thought we could subtract it automatically, so they user don't run into troubles if they don't set it up?

Reply:
That would be ideal but I'd be ok with having the user set this manually for now.

Reply:
If it's easy to do we should just do it of course.

Reply:
@knrt10 can you try implementing it? I think we could put it at Terraform template level, something like:
```
{{ if .Config.EncruptPodTraffic }}
network_mtu = {{ .Config.NetworkMTU }} - 40 # Wireguard overhead
{{- else }}
network_mtu = {{ .Config.NetworkMTU }}
{{- end }}
```

Reply:
Done, but it had to be implemented little differently, because we were setting default value in terraform, instead of that did that in go code, and it works